tools/fix_style.sh

```diff
@@ -13,9 +13,9 @@ fi
 for arg in "$@"
 do
     if [ -f $arg ]; then
-        clang-format-3.8 -i -style='{BasedOnStyle: google, ColumnLimit: 120}' $arg
+        clang-format-6.0 -i -style='{BasedOnStyle: google, ColumnLimit: 120}' $arg
     elif [ -d $arg ]; then
-        find $arg -iname '*.h' -o -iname '*.cpp' -o -iname '*.hpp' | xargs clang-format-3.8 -i -style='{BasedOnStyle: google, ColumnLimit: 120}'
+        find $arg -iname '*.h' -o -iname '*.cpp' -o -iname '*.hpp' | xargs clang-format-6.0 -i -style='{BasedOnStyle: google, ColumnLimit: 120}'
```

Comment:
Not related to this PR, but do we want to also change PX4/avoidance to clang-format-6.0?

Reply:
Eventually yes, since only 3.9 and above are available at Ubuntu 18.04.